src/main/java/tr/ogu/edu/school/schoolapp/service/ActTicketService.java

```diff
@@ -0,0 +1,53 @@
+package tr.ogu.edu.school.schoolapp.service;
+
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.stereotype.Service;
+
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import tr.ogu.edu.school.schoolapp.model.ActTicket;
+import tr.ogu.edu.school.schoolapp.repository.ActTicketRepository;
+
+@Service
+@RequiredArgsConstructor
+public class ActTicketService {
+    private final ActTicketRepository actTicketRepository;
+    private final PdfGeneratorService pdfGeneratorService;
+
+    @Transactional
+    public void createActTicket(ActTicket actTicket) throws FileNotFoundException {
+
+        if (actTicket != null){
+
+            // ActTicket objesinden gerekli parametreleri alıyoruz
+            Map<String, Object> parameters = preparePdfParameters(actTicket); 
+
+            // FIXME Daha sonra veritabanından çekilmeli
+            String templatePath = "jasper-report-templates/ticket.jrxml"; // Şablon dosyasının bulunduğu yer
+
+            // PDF üretme metodunu çağırıyoruz
+            String pdfFilePath = pdfGeneratorService.generatePdf(actTicket.getVerificationCode(), parameters, templatePath);
+            actTicket.setFilepath(pdfFilePath); 
+            actTicketRepository.save(actTicket);    // Kayıt yapılıyor
+
+        }
+        else{
+            // Bilet yok
+            throw new FileNotFoundException("Bilet bulunamadı");
```

Comment:
```suggestion
            throw new TicketRequiredException("Kaydedilecek bir bilet verisi gönderilmedi.");
```

benzeri bir Exception sınıfı oluşturularak gönderilmeli.